test/e2e/bootstrapper_test.go

```diff
@@ -125,26 +133,33 @@ func svcExists(t *testing.T, svcName string) bool {
 	}
 }
 
-// getSvcState gets the current state of the specified service. Requires administrator privileges
-func getSvcState(svcName string) (svc.State, error) {
+// getSvcInfo gets the current state and the fully qualified path of the specified service.
+// Requires administrator privileges
+func getSvcInfo(svcName string) (svc.State, string, error) {
```

Comment:
nit: Ideally have a struct around `svc` and write functions around it..

Reply:
Ideally I would prefer it too - but we should redesign the code to have better modularization for `svc` and `mgr` specific calls, so we decided not to push it as a part of this PR